respawn_altering/wizard

```diff
@@ -1,6 +1,5 @@
 Wizard- (5)
-- Can reduce or increase a person's respawn time by 2 phases during the night.
+- At night, the Wizard can perform a ritual spell on a respawning players body, this results in their respawn progress being adjusted by 2 phases. The Wizard can choose whether the adjustment is addition or subtraction (can become a negative number).
```

Comment:
While I like the flavour, this might be a bit too much, I feel like we should scale it a bit back and explain what actually happens

Reply:
it took me like 10 minutes to think of a short, concise way to explain it. if you can think of better phrasing, let me know

Reply:
Idk of better phasing, I guess it's good already. What does  Can become a negative number mean?

Reply:
if its 1 or 0 and you reduce it by 2 phases

Reply:
Ah okay, so they respawn as soon as the phase changes? Or is it instant? Shouldn't we instead cap it at 0?

Reply:
but then it would be impossible to use it on a role with 2 phase respawn time, which is why I added the negative part

Reply:
Alright, but does the negative mean it's instant or does it happen as soon as it can?

Reply:
Also shouldn't it be added to Healer too?

Reply:
would be instant respawn if it complete it, since "people respawn" comes after post-night during the day phase action order

Reply:
Alright sounds good and clear to me now. Thanks :D